dask/dataframe/core.py

```diff
@@ -7433,7 +7433,9 @@ def to_datetime(arg, meta=None, **kwargs):
 
 
 @wraps(pd.to_timedelta)
-def to_timedelta(arg, unit="ns", errors="raise"):
+def to_timedelta(arg, unit=None, errors="raise"):
+    if not PANDAS_GT_110 and not unit:
```

Comment:
I wonder if this should this be

```suggestion
    if not PANDAS_GT_110 and unit is None:
```

Can someone pass a falsey value for `unit` that is a valid input here?

Reply:
Thanks for the review!

> Can someone pass a falsey value for unit that is a valid input here?

An empty string? In which case we would want to raise an error, so your suggestion is the right approach!